opentracing-cdi/src/main/java/io/opentracing/contrib/cdi/internal/TracerInitializer.java

```diff
@@ -0,0 +1,60 @@
+package io.opentracing.contrib.cdi.internal;
+
+import io.opentracing.Tracer;
+import io.opentracing.contrib.tracerresolver.TracerResolver;
+import io.opentracing.util.GlobalTracer;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.Initialized;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import java.util.logging.Logger;
+
+/**
+ * This is not part of the "business" of this example, but can be used as an example of how to register a Tracer
+ * with the GlobalTracer. On some platforms, like Wildfly Swarm, this is done automatically for you.
+ *
+ * @author Juraci Paixão Kröhling
+ */
+public class TracerInitializer {
```

Comment:
I think the order of getting the tracer should be:

1. get tracer from CDI context
2. get from globalTracer
(3. use tracerresolver) - if we decide to depend on it